ISSUE_TEMPLATE.md

```diff
@@ -1,28 +1,65 @@
-# All
+# Issue
 
 _Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely._
 
-Measure of Completion:  
-Suggested Budget:
+**Content:**
```

Comment:
Please don't use **bold** markup where you mean a ...

### Header

Reply:
True.